src/main/kotlin/no/nav/medlemskap/services/aareg/AaRegClient.kt

```diff
@@ -92,23 +92,30 @@ class AaRegService(
     suspend fun hentArbeidsforhold(fnr: String, callId: String, fraOgMed: LocalDate? = null, tilOgMed: LocalDate? = null): List<Arbeidsforhold> {
         val arbeidsforhold = aaRegClient.hentArbeidsforhold(fnr, callId, fraOgMed, tilOgMed)
         val arbeidsgiver: List<AaRegOpplysningspliktigArbeidsgiver> = arbeidsforhold.map { it.arbeidsgiver }
-        val arbeidsgiversLand = hentArbeidsgiversLand(arbeidsgiver, callId)
 
-        return mapAaregResultat(arbeidsforhold, arbeidsgiversLand)
+        val arbeidsgiversLand = arbeidsgiver.associateBy(
```

Comment:
her så vil det bli mappet null. Har lyst til å gjøre noe med det, men er litt tom for idéer om hvordan jeg skal forhindre det. Tenkte å sjekke på arbeidsforholdtype og se om det er organisasjon i en foreach

er åpen for innspill

Reply:
vet ikke om jeg helt skjønner. hva blir null her?

Reply:
Kommentaren var ment for der arbeidsgiverenhetstypen blir hentet 😅 

Kan bli null dersom det ikke er noen orgnummer.

Reply:
samtidig kan det også bli null dersom det ikke er offentligident eller aktørid

Reply:
Når de gjelder edge-casene som evt ikke har noen ident i det hele tatt, så kan du kanskje gå veien om å lage en liste av Pair<String, Arbeidsgiver>, og så lage en map av lista? Noen typer lister kan inneholde null-verdier, og så kan du filtrere dem bort før du lager map'en. ?